index.html

```diff
@@ -354,10 +354,10 @@ <h2>
             I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
         </p>
         <div>
-            <form>
-                <input class="hover" placeholder="Full name">
-                <input class="hover" placeholder="Email address">
-                <textarea class="hover" placeholder="Enter text here"></textarea>
+            <form action="https://formspree.io/f/mqkozkka" method="post">
+                <input type="text" class="hover" maxlenght="30" placeholder="Full name" required>
+                <input type="email" class="hover" placeholder="Email address" required>
+                <textarea class="hover" maxlenght="500" placeholder="Enter text here" required></textarea>
```

Comment:
- [ ] You did a great job connecting your form to use the Formspree service as suggested by the previous reviewers! 🚀 However, there's one small thing that's preventing you from approving this project, your input fields (including the `<textarea>`) don't have a `name` attribute. This attribute is important because it is the one that let you identify each field when you send data. Please consider adding this! 🤓 🛠️ 

This is the feedback from `Formspree`:
![image](https://user-images.githubusercontent.com/77069011/207898774-30eee0a4-c44a-4c00-8c06-6b77e27d2fa4.png)

Here's an example, feel free to change the names as you wish! 👍🏼 

```suggestion
            <form action="https://formspree.io/f/mqkozkka" method="post">
                <input type="text" name="name" class="hover" maxlenght="30" placeholder="Full name" required>
                <input type="email" name="email" class="hover" placeholder="Email address" required>
                <textarea class="hover" name="message" maxlenght="500" placeholder="Enter text here" required></textarea>
```

Before submitting this pull request again, please make sure everything is working correctly! Cheers and happy coding 🍾